test/vt_object_tests.cpp

```diff
@@ -2975,3 +2975,265 @@ TEST(VIRTUAL_TERMINAL_OBJECT_TESTS, ObjectPointerTests)
 	EXPECT_TRUE(externalObject->get_attribute(static_cast<std::uint8_t>(ObjectPointer::AttributeName::Type), testValue));
 	EXPECT_EQ(testValue, static_cast<std::uint8_t>(VirtualTerminalObjectType::ObjectPointer));
 }
+
+TEST(VIRTUAL_TERMINAL_OBJECT_TESTS, AuxiliaryInputType1Tests)
+{
+	std::map<std::uint16_t, std::shared_ptr<VTObject>> objects;
+	auto auxiliaryInput = std::make_shared<AuxiliaryInputType1>();
+
+	run_baseline_tests(auxiliaryInput.get());
+	EXPECT_EQ(auxiliaryInput->get_object_type(), VirtualTerminalObjectType::AuxiliaryInputType1);
+
+	auxiliaryInput->set_function_type(AuxiliaryInputType1::FunctionType::Analogue);
+	EXPECT_EQ(AuxiliaryInputType1::FunctionType::Analogue, auxiliaryInput->get_function_type());
+
+	auxiliaryInput->set_input_id(200);
+	EXPECT_EQ(200, auxiliaryInput->get_input_id());
+
+	auxiliaryInput->set_input_id(254); // Max is 250
```

Comment:
```suggestion
	ASSERT_FALSE(auxiliaryInput->set_input_id(254)); // Max is 250
```
I think returning true/false would be nice, the docs currently doesn't state it caps at 250, eventhough it's in the standard it's nice to have here as well to avoid confusion

Reply:
Added! 🙂